Vagrantfile

```diff
@@ -146,6 +146,17 @@ Vagrant.configure("2") do |config|
         'securedrop:children' => %w(development),
       }
     end
+
+    # TODO: For some reason, the build VM is defaulting to using 1GB of RAM.
+    # It does not need this much RAM, and the staging environment has been
+    # causing issues (hangs and crashes) on developer machines that have <= 8GB
+    # RAM, so hopefully setting this to a smaller value will help with that
+    # issue. We should look into why this is being automatically allocated 1GB
+    # of RAM instead of the expected 512MB default.
```

Comment:
Best guess is that it's a bug in how the Vagrantfile declarations are resolved, since that's a notoriously difficult to predict process. The behavior only affects virtualbox—I cannot reproduce in libvirt, which always assigns 512MB to the build VM—so difficult to debug. Happy with the hardcoded solution, since it's explicit.